spec/requests/application_controller_spec.rb

```diff
@@ -547,6 +547,34 @@
         expect(response.headers["Cross-Origin-Opener-Policy"]).to eq("unsafe-none")
       end
     end
+
+    describe "when `cross_origin_unsafe_none_referrers` are present" do
+      before do
+        SiteSetting.cross_origin_opener_policy_header = "same-origin"
+        SiteSetting.cross_origin_opener_unsafe_none_referrers = "https://meta.discourse.org"
```

Comment:
I think we should test with more referrals instead of just one to ensure that the code in the controller is working against a list.